### PR TITLE
Add theme color support

### PR DESCRIPTION
**Pull request by timsueberkrueb**
See #14.
Note that the settings and a lot of the structure will change with `feature/extensions` (see #22).
But since there is no release yet, this shouldn't really matter.

### CODE REVIEW

File: src/core/settings/settings.h [hunk 1]
```diff
@@ -29,13 +29,21 @@
 
 #include "startconfig.h"
 #include "searchconfig.h"
+<<<<<<< e738755cea07dcb93be91605a6bab4a62d702c10
```
Comment (plfiorini) [inline src/core/settings/settings.h]: merge garbage here :D
please fix

File: src/ui/tabview/content/SettingsContent.qml [hunk 2]
```diff
@@ -159,6 +159,21 @@ TabContent {
                         }
                     }
                 }
+
+                TitleLabel {
+                    text: "Theme"
+                }
+
+                CheckBox {
+                    text: "Adapt to website theme colors"
```
Comment (plfiorini) [inline src/ui/tabview/content/SettingsContent.qml]: use qsTr()

File: src/core/models/tab.cpp [hunk 3]
```diff
@@ -29,6 +29,7 @@ Tab::Tab(QObject *parent, bool invalid)
 {
     m_uid = -1;
     m_canReload = true;
+    m_iconColor = QColor("transparent");
```
Comment (plfiorini) [inline src/core/models/tab.cpp]: Qt::transparent, I think it might be more efficient

File: src/ui/tabview/content/SettingsContent.qml [hunk 4]
```diff
@@ -211,6 +211,17 @@ TabContent {
                 }
 
                 ColumnLayout {
+                    CheckBox {
+                        text: "Adapt to website theme colors"
```
Comment (plfiorini) [inline src/ui/tabview/content/SettingsContent.qml:215]: qsTr

**Review by plfiorini — CHANGES_REQUESTED** [review]

File: src/core/settings/settings.h (hunk 1 above)
Comment (timsueberkrueb) [reply src/core/settings/settings.h]: oh no xD

Comment (timsueberkrueb) [reply src/core/settings/settings.h]: Will fix

Comment (timsueberkrueb) [reply src/core/settings/settings.h]: Fixed.

File: src/ui/tabview/content/SettingsContent.qml (hunk 4 above)
Comment (timsueberkrueb) [reply src/ui/tabview/content/SettingsContent.qml:215]: See #13 :) Right now, no string in the browser is translatable.

File: src/core/models/tab.cpp (hunk 3 above)
Comment (timsueberkrueb) [reply src/core/models/tab.cpp]: Done

**Review by plfiorini — APPROVED** [review]

**Pull request merged**